spec/jobs/scheduled_publishing_job_spec.rb

```diff
@@ -10,12 +10,41 @@
       publish_service = double(:publish_service, publish: nil)
 
       expect(PublishService).to receive(:new).with(edition) { publish_service }
+      message_delivery = double(ActionMailer::MessageDelivery, deliver_later: nil)
 
       expect(publish_service)
         .to receive(:publish)
         .with(user: user, with_review: edition.status.details.reviewed)
 
-      ScheduledPublishingJob.new.perform(edition.id)
+      expect(ScheduledPublishMailer).to receive(:publish_email).with(edition, user)
+                                                               .and_return(message_delivery)
+
+      ScheduledPublishingJob.perform_now(edition.id)
+    end
+
+    it "calls ScheduledPublishMailer for each editor of the edition" do
+      editor_one = create(:user, email: "someone@example.com")
+      metadata_revision = create(:metadata_revision, scheduled_publishing_datetime: Time.current)
+      editor_two = create(:user, email: "someone-else@example.com")
+      revision = create(:revision,
+                        created_by: editor_two,
+                        metadata_revision: metadata_revision)
```

Comment:
I think you can set `scheduled_publishing_datetime` here rather than needing to create a metadata_revision.

Reply:
You're correct!